packages/block-editor/src/components/inserter/index.native.js

```diff
@@ -2,19 +2,49 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Dropdown, ToolbarButton, Dashicon } from '@wordpress/components';
+import { Dropdown, ToolbarButton, Dashicon, Picker } from '@wordpress/components';
 import { Component } from '@wordpress/element';
 import { withSelect } from '@wordpress/data';
 import { compose, withPreferredColorScheme } from '@wordpress/compose';
-import { getUnregisteredTypeHandlerName } from '@wordpress/blocks';
+import { isUnmodifiedDefaultBlock } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 import InserterMenu from './menu';
 
-const defaultRenderToggle = ( { onToggle, disabled, style } ) => (
+const addBeforeOption = {
```

Comment:
Not sure it's worth it having them module scoped, they could be defined inside `getInsertionOptions`

Reply:
👍